### PR TITLE
Enable devicePreference on Electron.js samples

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@mingmingtasd, @fujunwei, PTAL, thanks!

File: node/examples/electron/image_classification/main.js [hunk 1]
```diff
@@ -18,14 +18,22 @@ function createWindow() {
     }
   })
 
-  // Load the index.html with 'numRuns' to run inference multiple times.
+
   let url = `file://${__dirname}/image_classification/index.html`
+  let numRunsParam, deviceParam
```
Comment (mingmingtasd) [inline node/examples/electron/image_classification/main.js:23]: There are some duplicated codes for these samples, should we combine them into one function like ParseAndCheckParams()?

File: node/examples/electron/image_classification/main.js [hunk 2]
```diff
@@ -18,14 +18,22 @@ function createWindow() {
...
   for (let argv of process.argv) {
-    if (argv.startsWith("numRuns")) {
-      url += '?' + argv
-      break
+    if (argv.startsWith("numRuns=") && !numRunsParam) {
+      // Load the index.html with 'numRuns' to run inference multiple times.
+      numRunsParam = argv
```
Comment (mingmingtasd) [inline node/examples/electron/image_classification/main.js:27]: These codes may look confused. How about using a list/array to store these params, then generating the url by it?

Comment (mingmingtasd) [reply node/examples/electron/image_classification/main.js:27]: And I think it's no necessary to check whether users set duplicated params.

Comment (Honry) [reply node/examples/electron/image_classification/main.js]: Thanks @mingmingtasd, done in latest commit. PTAL.

File: node/examples/electron/image_classification/main.js (hunk 1 above)
Comment (Honry) [reply node/examples/electron/image_classification/main.js]: We are going to migrate all standalone webnn samples into one folder to reduce duplicated codes, https://github.com/otcshare/webnn-native/issues/796

So let's optimize this there.

**Review by fujunwei — APPROVED** [review]
LGTM.

File: node/examples/electron/image_classification/main.js [hunk 3]
```diff
@@ -18,14 +18,26 @@ function createWindow() {
     }
   })
 
-  // Load the index.html with 'numRuns' to run inference multiple times.
+
   let url = `file://${__dirname}/image_classification/index.html`
+  let params = {numRuns: "", device: ""}
+  let urlPrex = ""
```
Comment (mingmingtasd) [inline node/examples/electron/image_classification/main.js:24]: urlSuffix?

File: node/examples/electron/image_classification/main.js (hunk 1 above)
Comment (mingmingtasd) [reply node/examples/electron/image_classification/main.js]: OK,got it, thanks!

**Review by mingmingtasd — APPROVED** [review]
LGTM, thanks!

**Comment by fujunwei** [comment]
Merge it.

**Pull request merged**